src/search/data_loader.cpp

```diff
@@ -103,7 +103,7 @@ std::vector<ahItem*> CDataLoader::GetAHItemsToCategory(uint8 AHCategoryID, int8*
     const char* fmtQuery = "SELECT item_basic.itemid, item_basic.stackSize, COUNT(*)-SUM(stack), SUM(stack) "
         "FROM item_basic "
         "LEFT JOIN auction_house ON item_basic.itemId = auction_house.itemid AND auction_house.buyer_name IS NULL "
-        "LEFT JOIN item_armor ON item_basic.itemid = item_armor.itemid "
+        "LEFT JOIN item_equip ON item_basic.itemid = item_armor.itemid "
```

Comment:
Should this have `= item_armor.idemid`, or should it be changed to `= item_equip.itemid`?

Reply:
good find and i found another, search doesnt search very well

Reply:
You could use something like `git grep item_armor` instead of depending on whatever search you used.